terraform/environments/staging/main.tf

```diff
@@ -22,13 +22,14 @@ module "staging" {
   ecr_repositories = [ "gql-server" ]
 
   # EKS configuration
-  eks_worker_instance_type = "t2.small"
-  eks_worker_asg_min_size  = 2
-  eks_worker_asg_max_size  = 6
+  eks_cluster_version      = "1.15"
+  eks_worker_instance_type = "t3.medium"
```

Comment:
t2.small doesn't have enough network interfaces to be very useful even on staging.